app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java

```diff
@@ -0,0 +1,10 @@
+package com.example.concordia_campus_guide.Models;
+
+//TODO: Make sure these strings correspond exactly to the PointTypes
```

Comment:
Before completing this, we will need to change the PointType into a typeDef ([for better performance](https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3)). The todo is created as a reminder (an improvement issue will probably be created too).

Reply:
Yeah create an improvement issue asap in order to not forget

Reply:
Done! [#147]